.github/workflows/build-and-deploy.yml

```diff
@@ -0,0 +1,43 @@
+name: "Build and Deploy"
+on:
+  push: {}
+  pull_request: {}
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - uses: cachix/install-nix-action@v23
+      with:
+        nix_path: nixpkgs=channel:nixos-unstable
```

Comment:
Is that needed at all, or is there something about Cachix that relies on channels? Ideally we'd have NIX_PATH empty or unset.

Reply:
Not sure, I copied that from your existing config! https://github.com/NixOS/nix.dev/blob/master/.github/workflows/build.yml#L16

Reply:
Let's keep this a verbatim copy then, and make improvementsin follow-up PR.s